core/src/Kokkos_Half.hpp

```diff
@@ -1254,6 +1260,19 @@ struct Kokkos::Experimental::Impl::max_exponent_helper<
 
 ////////////// BEGIN BHALF_T (bfloat16) limits //////////////
 #if defined(KOKKOS_BHALF_T_IS_FLOAT) && !KOKKOS_BHALF_T_IS_FLOAT
+/// \brief: Infinity
+///
+///
+/// Bfloat16 encoding:
+///             [s  e  e  e  e  e  e  e  e f f f f f f f]
+///             [0  1  1  1  1  1  1  1  1 0 0 0 0 0 0 0]
+/// bit index:   15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0
```

Comment:
Looks like you need an extra space somewhere to align this properly?
If it looks good in the documentation then its fine.